deployments/operator/samples/nfd_v1_nodefeaturediscovery.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: nfd.openshift.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  name: nfd-instance
+  namespace: openshift-nfd
+spec:
+  extraLabelNs:
+    - sgx.intel.com
+  resourceLabels:
+    - sgx.intel.com/epc
+  operand:
+    image: quay.io/openshift/origin-node-feature-discovery:4.10
+    imagePullPolicy: Always
+    servicePort: 12000
+  workerConfig:
+    configData: |
+      core:
+        sleepInterval: 60s
+  customConfig:
+    configData: |
+      - name: "intel.sgx"
+        labels:
+          "intel.feature.node.kubernetes.io/sgx": "true"
+        matchFeatures:
+          - feature: cpu.cpuid
+            matchExpressions:
+              SGX: {op: Exists}
+              SGXLC: {op: Exists}
+          - feature: cpu.sgx
+            matchExpressions:
+              enabled: {op: IsTrue}
+          - feature: kernel.config
+            matchExpressions:
+              X86_SGX: {op: Exists}
```

Comment:
We are maintaining `deployments/nfd/overlays/node-feature-rules` for this functionality and I'd like you to move to use that too.

Reply:
moved as discussed offline.